marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/mastering/validateMergingStep.api

```diff
@@ -0,0 +1,15 @@
+{
+  "functionName": "validateMerge",
```

Comment:
For consistency, this should be the same as the endpoint name.

Reply:
Also, should this PR include updating the MasteringService interface? Can run "gradle generatemasteringserviceinterface"

Reply:
added the middle-tier logic